tactics/eqschemes.ml

```diff
@@ -256,7 +257,7 @@ let const_of_scheme kind env handle ind ctx =
   let sym_scheme = match local_lookup_scheme handle kind ind with Some cst -> cst | None -> assert false in
   with_context_set ctx (UnivGen.fresh_global_instance env sym_scheme)
 
-let build_sym_involutive_scheme env handle ind =
+let build_sym_involutive_scheme env handle ind _ =
```

Comment:
I don't know what is the standard in Ocaml, but I personally think it is much better to add an eta-expansion when the function needs to be applied like `fun x y _ -> f x y` than to add an unnamed variable at the definition `f x y _`.
It is not the end of the world as `build_sym_involutive_scheme` is not exposed in the `.mli` file, but it still makes it harder to maintain the code as you don't get why there is an extra-variable until you find where it is applied